src/SaveHandler/MongoDB.php

```diff
@@ -10,10 +10,10 @@
 
 namespace Zend\Session\SaveHandler;
 
-use Mongo;
-use MongoClient;
-use MongoCollection;
-use MongoDate;
+use MongoDB\BSON\Binary;
+use MongoDB\BSON\UTCDatetime;
+use MongoDB\Client as MongoClient;
```

Comment:
It's only available in mongo php library. We don't need this. We can do everything with the raw extension here, imho.